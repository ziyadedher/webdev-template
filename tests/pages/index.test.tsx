/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react";

import Index from "../../pages";

describe("Index", () => {
  test("renders", () => {
    const result = render(<Index />);
    expect(result).toMatchSnapshot();
  });
});
