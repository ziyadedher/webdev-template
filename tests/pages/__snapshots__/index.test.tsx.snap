// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container flex flex-row justify-center items-center mx-auto h-screen"
      >
        <div
          class="text-center"
        >
          <h1
            class="text-6xl text-gray-700"
          >
            Hello, world!
          </h1>
          <h2
            class="pt-4"
          >
            This is
             
            <a
              class="text-blue-500"
              href="https://twitter.com/ziyadedher"
              rel="noreferrer"
              target="_blank"
            >
              @ziyadedher
            </a>
            ’s webapp template.
          </h2>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container flex flex-row justify-center items-center mx-auto h-screen"
    >
      <div
        class="text-center"
      >
        <h1
          class="text-6xl text-gray-700"
        >
          Hello, world!
        </h1>
        <h2
          class="pt-4"
        >
          This is
           
          <a
            class="text-blue-500"
            href="https://twitter.com/ziyadedher"
            rel="noreferrer"
            target="_blank"
          >
            @ziyadedher
          </a>
          ’s webapp template.
        </h2>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
